test/Microsoft.Azure.Devices.Edge.Test/Provisioning.cs

```diff
@@ -67,5 +74,61 @@ public async Task DpsSymmetricKey()
             Context.Current.DeleteList.TryAdd(registrationId, device.Expect(() => new InvalidOperationException(
                 $"Device '{registrationId}' should have been created by DPS, but was not found in '{this.iotHub.Hostname}'")));
         }
+
+        [Test]
+        public async Task DpsX509()
+        {
+            (string, string, string) rootCa =
+                        Context.Current.RootCaKeys.Expect(() => new InvalidOperationException("Missing root CA keys"));
+            string caCertScriptPath =
+                        Context.Current.CaCertScriptPath.Expect(() => new InvalidOperationException("Missing CA cert script path"));
+            string idScope = Context.Current.DpsIdScope.Expect(() => new InvalidOperationException("Missing DPS ID scope"));
+            string registrationId = this.GetRegistrationId();
+
+            Assert.IsTrue(File.Exists(Path.Combine(caCertScriptPath, "certGen.sh")), "CA Certification script does not exist. Path: " + $"{caCertScriptPath}" );
+
+            CertificateAuthority ca;
+            CancellationToken token = this.TestToken;
+
+            using (var cts = new CancellationTokenSource(Context.Current.SetupTimeout))
```

Comment:
We already have a `CancellationTokenSource` for this test; it's created in `BaseFixture` and it's token is already set to local variable `token` above. You can remove this `using` statement and the braces, and replace instances of `caToken` with `token`.

Reply:
remove `caToken`